ksql-common/src/main/java/io/confluent/ksql/util/KsqlConstants.java

```diff
@@ -25,6 +25,9 @@
       "WINDOW_CHANGE_LOG_ADDITIONAL_RETENTION";
   public static final String SINK_TIMESTAMP_COLUMN_NAME = "TIMESTAMP";
 
+  public static final String SCHEMA_REGISTRY_VALUE_SUFFIX = "-value";
```

Comment:
This seems to be used in only to append `-value` to the kafka topic name when fetching schema metadata. I still don't understand what purpose adding this suffix serves.

Reply:
In Schema Registry for each topic we have two schemas, one for key and one for value. Schema registry adds these suffixes automatically. When we create a new stream/table we should register the schema in schema registry and since the schema is for the value we should add `-value` suffix.

Reply:
So in this case schema registry will not append another `-value`?

Reply:
Also, a comment about this in the code would be hugely beneficial.